src/EnergyPlus/GroundHeatExchangers.cc

```diff
@@ -558,12 +539,17 @@ GLHEVertProps::GLHEVertProps(EnergyPlusData &state, std::string const &objName,
 std::shared_ptr<GLHEVertProps> GetVertProps(EnergyPlusData &state, std::string const &objectName)
 {
     // Check if this instance of this model has already been retrieved
-    for (auto &thisProp : state.dataGroundHeatExchanger->vertPropsVector) {
-        // Check if the type and name match
-        if (objectName == thisProp->name) {
-            return thisProp;
-        }
-    }
```

Comment:
[src/EnergyPlus/GroundHeatExchangers.cc:563]:(style),[useStlAlgorithm],Consider using std::find_if algorithm instead of a raw loop.